NachoSpigot-Server/src/main/java/dev/cobblesword/nachospigot/Nacho.java

```diff
@@ -21,17 +22,28 @@
 
     private final Set<PacketListener> packetListeners = Sets.newConcurrentHashSet();
     private final Set<MovementListener> movementListeners = Sets.newConcurrentHashSet();
+    public static HitDetection hitDetectionThread;
+    public static Knockback knockbackThread;
 
     public Nacho() {
         INSTANCE = this;
 
         AsyncExplosions.initExecutor(NachoConfig.useFixedPoolForTNT, NachoConfig.fixedPoolSize);
 
+        if (NachoConfig.asyncHitDetection) {
+            this.hitDetectionThread = new HitDetection("Hit Detection Thread");
```

Comment:
How is this string argument used? It's passed down to AsyncThread, which ignores it.